frontend/appflowy_flutter/lib/mobile/presentation/bottom_sheet/show_mobile_bottom_sheet.dart

```diff
@@ -3,6 +3,24 @@ import 'package:appflowy/plugins/base/drag_handler.dart';
 import 'package:flowy_infra_ui/flowy_infra_ui.dart' hide WidgetBuilder;
 import 'package:flutter/material.dart';
 
+extension BottomSheetPaddingExtension on BuildContext {
+  /// Calculates the total amount of space that should be added to the bottom of
+  /// a bottom sheet
+  double bottomSheetPadding({
+    bool ignoreViewPadding = true,
+  }) {
+    final mediaQuery = MediaQuery.of(this);
+    double bottom = 0.0;
+    if (!ignoreViewPadding) {
+      bottom += mediaQuery.viewPadding.bottom;
+    }
+    // for screens with 0 view padding, add some even more space
+    bottom += mediaQuery.viewPadding.bottom == 0 ? 28.0 : 16.0;
+    bottom += mediaQuery.viewInsets.bottom;
+    return bottom;
```

Comment:
Prefer `.sizeOf` and similar static methods.
```suggestion
    final viewPadding = MediaQuery.viewPaddingOf(this);
    final viewInsets = MediaQuery.viewInsetsOf(this);
    double bottom = 0.0;
    if (!ignoreViewPadding) {
      bottom += viewPadding.bottom;
    }
    // for screens with 0 view padding, add some even more space
    bottom += viewPadding.bottom == 0 ? 28.0 : 16.0;
    bottom += viewInsets.bottom;
    return bottom;
```